tests/System.Buffers.Experimental.Tests/MemoryTests.cs

```diff
@@ -90,5 +90,161 @@ static void TestLifetime(OwnedBuffer<byte> owned)
                 var span = copyStoredForLater.Span;
             });
         }
+
+        [Fact]
+        public unsafe void IsSliceOf_PointerVersusArraySpansNeverMatch()
+        {
+            ulong[] data = new ulong[8];
+            fixed (ulong* ptr = data)
+            {
+                Span<ulong> arrayBased = data;
+                Span<ulong> pointerBased = new Span<ulong>(ptr, 8);
+
+                // comparing array to pointer says "no"
+                Assert.False(arrayBased.IsSliceOf(pointerBased));
```

Comment:
Why would this be no?
The arrayBased Span and pointerBased Span point to the same memory.